GenderPayGap.WebUI/Views/ReportStarting/ReportStarting.cshtml

```diff
@@ -0,0 +1,84 @@
+﻿@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using GovUkDesignSystem
+@model GenderPayGap.WebUI.Models.ReportStarting.ReportStartingViewModel
+
+@{
+    ViewBag.Title = $"Report gender pay gap data - reporting year  - Gender pay gap service"; // TODO: insert snapshot date

```

Comment:
I think this title is missing the actual year - other report pages include this information (is this what the TODO is talking about?)